### ISSUE TITLE
`<schemaSpec>/@source` needs a "tei:none" value and should be mandatory

### ISSUE THREAD

**Issue by martindholmes** [TEIC/TEI#2591]
Coming from the ATOP work with @HelenaSabel and @sydb:

The `@source` attribute on `<schemaSpec>` is used to specify which TEI base odd version you want your schema to be compiled against (or the URL of another base odd at a specified URL). When it is not present, we assume the value of tei:current, the last stable release of TEI.

However, as ODD is increasingly used for non-TEI schema descriptions (something we support and encourage), it is increasingly common for the base TEI ODD not to be needed or wanted; in such cases, it is retrieved and processed pointlessly, and might even result in unexpected interference in the processing of an ODD.

For this reason, we propose two things:

1. We should add and document a new value of `@source="tei:none"`, meaning that no base TEI ODD is to be retrieved at all. This simplifies and streamlines processing in cases where no base ODD is required.
2. We should make `@source` on `<schemaSpec>` mandatory, so that no-one omits it by accident; it is all too easy to be taken by surprise when a new release of TEI suddenly results in changes to your schema that you did not expect, just because you omitted `@source` and so "tei:current" was assumed.

**Comment by peterstadler** [comment]
I agree with 1. and 2. but would recommend a different value for `@source` to declare that this ODD is not based on anything else. Since this is not a TEI feature, I'd try to avoid the "tei" prefix and simply label it as "no-source" or "odd:origin" …
